state/txn.go

```diff
@@ -333,7 +334,7 @@ func (txn *Txn) GetState(addr types.Address, key types.Hash) types.Hash {
 		}
 	}
 
-	// Under which condition is this called?
+	// Get account state from trie tree
```

Comment:
Can you please elaborate a bit here on what's the difference between `trie tree` and `radix tree` up on the line 327?

Reply:
Also, try to include a comment like: `If the object was not found in the radix trie (due to reasons XXX), we fetch it from the trie tre...`

In simpler terms, we should try to comment what is the logic behind these if branches.